README.md

```diff
@@ -75,6 +75,18 @@ npm run build:fast
 
 See also: [Contributing](#Contributing)
 
+### Other First-Time Build Requirements
+
+- [node-gyp](https://github.com/nodejs/node-gyp) Requirements
```

Comment:
I believe this is only required if you build certain server packages that requires some binaries and that binaries are not available for the node version you have?    Can we be more precise in where this is needed?

Reply:
I believe you're right. I ran into this running a fully-symlinked fluid-build, but the point of failure for me was in the server packages. I'll add some more notes here.

Reply:
I went ahead and moved the node-gyp specific notes into the Routerlicious README, and updated these docs to be expressed in terms of the workflows that include it.